www/js/coolpoints-util.js

```diff
@@ -37,7 +47,7 @@ function updateAnimation(id, animationName) {
   el.classList.add(animationName);
   Promise.all(
     el.getAnimations({ subtree: true }).map((animation) => animation.finished)
-  ).then(() => el.classList.remove(animationName));
+  ).then(() => el.classList.remove(animationName)).catch((error) => console.log(error));
```

Comment:
nit: remove (unless you really wanna know when a target animation is cancelled. See comment below

Reply:
Fixed with the latest commit